code/modules/mining/drilling/distribution.dm

```diff
@@ -178,7 +178,7 @@ Deep minerals:
 								target_turf.resources["silver"] =     rand(RESOURCE_LOW_MIN,RESOURCE_LOW_MAX)
 								target_turf.resources["uranium"] =    rand(RESOURCE_LOW_MIN,RESOURCE_LOW_MAX)
 								target_turf.resources["diamond"] =    0
-								target_turf.resources["phoron"] =     0
```

Comment:
Read up on how assoc lists work. It's a constant key, it's not going to fail to magically compile just because you don't like the word phoron.

'magic Bay-mining-code' my arse.